### DISCUSSION TITLE
[Feature]: Option to hide album content from main view

### DISCUSSION THREAD

**Discussion by The-Noah** [immich-app/immich#1645] [Feature Request]
### Feature detail

It'd be awesome if each album had a setting to hide its contents from the main view. That way I can have all my memes in an album and they don't take up space on my main view.

### Platform

Web

**Comment by NextBlaubeere** [comment]
This feature would be the only feature left I really miss on a daily basis.

Hiding my "memes" from the main mainline would be awesome!

**Comment by bo0tzz** [comment]
Closing this as a duplicate of #2451, as that has more detail/discussion.
